realsense2_camera/package.xml

```diff
@@ -12,20 +12,30 @@
 
   <author email="sergey.dorodnicov@intel.com">Sergey Dorodnicov</author>
   <author email="doron.hirshberg@intel.com">Doron Hirshberg</author>
+  
   <buildtool_depend>catkin</buildtool_depend>
-  <depend>image_transport</depend>
+  
   <depend>cv_bridge</depend>
-  <depend>nav_msgs</depend>
-  <depend>nodelet</depend>  
+  <depend>diagnostic_updater</depend>
+  <depend>ddynamic_reconfigure</depend>
   <depend>genmsg</depend>
+  <depend>image_transport</depend>
+  <depend>nav_msgs</depend>
+  <depend>nodelet</depend>
   <depend>roscpp</depend>
   <depend>sensor_msgs</depend>
   <depend>std_msgs</depend>
-  <depend>message_runtime</depend>
   <depend>tf</depend>
-  <depend>ddynamic_reconfigure</depend>
-  <depend>diagnostic_updater</depend>
-  <depend>librealsense2</depend>
+  
+  <build_depend>librealsense2-dbg</build_depend>
+  <build_depend>librealsense2-dev</build_depend>
```

Comment:
I'm pulling these packages from the amazon debian repos, and you need the `-dev` to build realsense-ros

Reply:
As for now, the packages for librealsense are not available as debian packages in ROS. Therefor it's on hold until they are accepted.

Reply:
We build realsense-ros in an internal CI system that requires the dependencies are straightened out in package.xml . Would you consider taking in these changes even if they're not used for upstream ROS builds (yet)?

Reply:
Here is my concern: Take a look at the .travis.yaml file we currently use. Will it work if I added the dependencies while librealsense is still not available?

Reply:
Ping?